lib/document.js

```diff
@@ -10,36 +9,10 @@ export default ({ head, css, html, data, dev, staticMarkup, cdn }) => {
     </head>
     <body>
       <div id='__next' dangerouslySetInnerHTML={{ __html: html }} />
-      {staticMarkup ? null : <script dangerouslySetInnerHTML={{ __html: '__NEXT_DATA__ = ' + htmlescape(data) }} />}
-      {staticMarkup ? null : createClientScript({ dev, cdn })}
+      {staticMarkup ? null : <script dangerouslySetInnerHTML={{
+        __html: `__NEXT_DATA__ =${htmlescape(data)}; module={};`
```

Comment:
I had to introduce this `module={}` to run the main.js.
It added `module.exports` and I couldn't figure out why. I suppose, that's something next's webpack plugins does.

Anyway, this fixes that and I think @nkzawa could suggest a proper fix if this is ugly :)